spec/lib/rtfm_spec.rb

```diff
@@ -103,30 +106,31 @@
   should not change the test results for the richmond comment blocks
 =end
     end
+
   end
 
   describe '.emit' do
+
     let! (:input_result) { subject.scan dir }
 
     before(:each) do
       input_result.output.each_pair do |file, lines|
-        if File.exists? file
-          File.delete file
-        end
+        File.delete file if File.exists? file
       end
     end
 
     let!(:emit_result) { subject.emit input_result.output }
 
     describe 'files created' do
+
       it 'emit_result is not nil' do
         expect(emit_result).to_not be nil
       end
 
       it 'creates the output files' do
-        input_result.output.keys.each {|file|
+        input_result.output.keys.each do |file|
```

Comment:
More personal preference... but I think a generally-accepted practice is to use curly braces when the block can fit on one line, and use do-end if you hit the enter key.